scala-package/macros/src/main/scala/org/apache/mxnet/NDArrayMacro.scala

```diff
@@ -91,69 +114,74 @@ private[mxnet] object NDArrayMacro {
     val newNDArrayFunctions = {
       if (isContrib) ndarrayFunctions.filter(
         func => func.name.startsWith("_contrib_") || !func.name.startsWith("_"))
-      else ndarrayFunctions.filterNot(_.name.startsWith("_"))
+      else ndarrayFunctions.filterNot(f => f.name.startsWith("_") &&
+        !f.name.startsWith("sample_") && !f.name.startsWith("random_"))
```

Comment:
You should not try to avoid `sample_` and `random_` from generation:
[This](http://mxnet.incubator.apache.org/api/python/ndarray/ndarray.html#mxnet.ndarray.random_uniform) called by operator `random_uniform`.
and [This](http://mxnet.incubator.apache.org/api/python/ndarray/random.html#mxnet.ndarray.random.uniform) called by operator `_random_uniform`